### PR TITLE
chore: add missing tags for posts

### PR DESCRIPTION
**Pull request by bahildebrand**
A large number of our posts don't have tags, and
are therefore hard to discover. Go back and add
tags to these posts.

### CODE REVIEW

**Comment by bahildebrand** [comment]
Current dependencies on/for this PR:
* master
  * **PR #340** <a href="https://app.graphite.dev/github/pr/memfault/interrupt/340" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/memfault/interrupt/340?utm_source=stack-comment).

File: _posts/2022-08-03-installing-gdb.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Tools we use: installing GDB for ARM"
 description: Demonstrate a few methods of installing gdb for ARM
 author: noah
 image: img/installing-gdb/cover.png # 1200x630
+tags: [ARM, gdb]
```
Comment (noahp) [inline _posts/2022-08-03-installing-gdb.md:6]: should tags be all lowercase?

**Review by noahp — APPROVED** [review]
nice!

File: _posts/2022-08-03-installing-gdb.md (hunk 1 above)
Comment (bahildebrand) [reply _posts/2022-08-03-installing-gdb.md:6]: We had some previous tags that were uppercase, but I think I was just being lazy. I'll change them to all be lowercase.

**Review by tyhoff — APPROVED** [review]
Love it - something that just needed to be done. Appreciate it!

**Pull request merged**